packages/x-data-grid/src/tests/rows.DataGrid.test.tsx

```diff
@@ -750,7 +750,7 @@ describe('<DataGrid /> - Rows', () => {
             width={100}
           />,
         );
-        expect(document.querySelectorAll('.MuiDataGrid-cell')).to.have.length(2);
+        expect(document.querySelectorAll('.MuiDataGrid-cell')).to.have.length(3);
```

Comment:
```suggestion
        expect($$(`.${gridClasses.cell}:not(.${gridClasses.cellEmpty})`)).to.have.length(2);
```
To make it more clear